### ISSUE TITLE
Modify bool_column to toggle boolean values

### ISSUE THREAD

**Issue by ldlsegovia** [platanus/activeadmin_addons#148]
If `interactive: true` option is present...

```ruby
  bool_column :paid, interactive: true
```

after clicking on the label (x)...

<img width="84" alt="screen shot 2017-09-03 at 12 54 02 pm" src="https://user-images.githubusercontent.com/3026413/30004508-feb39c04-90a6-11e7-845c-425a112a60c9.png">

it should to show something this:

<img width="219" alt="screen shot 2017-09-03 at 12 57 28 pm" src="https://user-images.githubusercontent.com/3026413/30004529-7bb40cca-90a7-11e7-8c74-40e1521c3a9c.png">

- This feature should be available in `bool_colum` and not in `bool_row`.
- It should be ajax.

**Comment by ldlsegovia** [comment]
https://github.com/platanus/activeadmin_addons/blob/master/docs/toggle_bool.md
